elasticapm/contrib/serverless/aws.py

```diff
@@ -50,11 +51,67 @@
 logger = get_logger("elasticapm.serverless")
 
 COLD_START = True
+INSTRUMENTED = False
 
-_AnnotatedFunctionT = TypeVar("_AnnotatedFunctionT")
+_AWSLambdaContextT = TypeVar("_AWSLambdaContextT")
 
 
-class capture_serverless(object):
+def capture_serverless(func: Optional[callable] = None, **kwargs) -> callable:
+    if not func:
+        # This allows for `@capture_serverless()` in addition to
+        # `@capture_serverless` decorator usage
+        return functools.partial(capture_serverless, **kwargs)
+
+    if kwargs:
+        warnings.warn(
+            PendingDeprecationWarning(
```

Comment:
Open question as to whether this needs to wait 18 months. I don't think anyone was using this with kwargs, it makes *way* more sense to use env variables in lambda environments, but it technically will be a breaking change. 🤷‍♂️